src/ripple/app/ledger/impl/ConsensusImp.h

```diff
@@ -0,0 +1,131 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2012, 2013 Ripple Labs Inc.
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose  with  or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#ifndef RIPPLE_APP_LEDGER_IMPL_CONSENSUSIMP_H_INCLUDED
+#define RIPPLE_APP_LEDGER_IMPL_CONSENSUSIMP_H_INCLUDED
+
+#include <BeastConfig.h>
+#include <ripple/app/ledger/Consensus.h>
+#include <ripple/app/ledger/LedgerConsensus.h>
+#include <ripple/app/misc/FeeVote.h>
+#include <ripple/protocol/STValidation.h>
+#include <ripple/shamap/SHAMap.h>
+#include <beast/utility/Journal.h>
+
+namespace ripple {
+
+/** Implements the consensus process and provides inter-round state. */
+class ConsensusImp
```

Comment:
What's the difference between a `Consensus` and a `LedgerConsensus`? They sound quite similar.